cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -40,7 +44,14 @@ class Robot(BaseSalesforceTask):
             "required": False,
         },
         "options": {
-            "description": "A dictionary of options to robot.run method.  See docs here for format.  NOTE: There is no cci CLI support for this option since it requires a dictionary.  Use this option in the cumulusci.yml when defining custom tasks where you can easily create a dictionary in yaml."
+            "description": "In simple cases this can be specified on the command line using "
+            "name:value,name:value syntax. More commplex cases can be specified "
+            "in cumuuluci.yml using YAML dictionary syntax.",
+            "required": False,
+        },
+        "outputdir": {
+            "description": "The directory in which to place HTML and XML output",
```

Comment:
This is slightly inaccurate, there could be other types of files there, such as screenshots. How about "the directory where robotframework will  write all output"?